raiden-cli/src/utils/formatting.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { RaidenChannel } from 'raiden-ts';
+import { ApiChannel } from '../types';
+
+export function transformChannelFormatSdkToApi(channel: RaidenChannel): ApiChannel {
```

Comment:
I like pure functions, no bind if not needed

Reply:
:heavy_check_mark: